SendWelcomeMessagesActivity/index.ts

```diff
@@ -13,14 +15,17 @@ import {
   setFetchTimeout,
   toFetch
 } from "italia-ts-commons/lib/fetch";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
 import { Millisecond } from "italia-ts-commons/lib/units";
 
 // HTTP external requests timeout in milliseconds
 const DEFAULT_REQUEST_TIMEOUT_MS = 10000;
 
 // Needed to call notifications API
 const publicApiUrl = getRequiredStringEnv("PUBLIC_API_URL");
```

Comment:
accordingto this change publicApiUrl may now contain the API gateway url OR the internal function URL; the same variable is used for different things according to the value of publicApiKey. this is error prone because you can end up with invalid configurations (ie. apiKey = empty, publicApiUrl = API gateway url)